images/kubevirt-infra-prow/publish_multiarch_prow_images.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/bash -x
+
+BAZEL_VERSION=3.0.0
+PROW_COMMIT=master
+repo=kubevirtci
+archs=(arm64 ppc64le)
+binaries=(clonerefs initupload entrypoint sidecar)
+
+main() {
+    local build_only
+    local tag
+    while getopts "bh" opt; do
+        case "$opt" in
+            b)
+                build_only=true
+                ;;
+            h)
+                help
+                exit 0
+                ;;
+            *)
+                echo "Invalid argument: $opt"
+                help
+                exit 1
+        esac
+    done
+    shift $((OPTIND-1))
+    tag=$(get_image_tag)
+    build_binaries
+    build_images
+    [[ $build_only ]] && return
+    push_images
+    push_manifest
+}
+
+help() {
+    cat <<EOF
+    Usage:
+        ./publish_multiarch_prow_images.sh [OPTIONS]
+
+    Build and publish mutliarch prow utility images.
+
+    OPTIONS
+        -h  Show this help message and exit.
+        -b  Only build the image and exit. Do not publish the built image.
+EOF
+}
+
+get_base_image() {
+    if [ $1 == "arm64" ]; then
+        echo "arm64v8/alpine"
+    elif [ $1 == "ppc64le" ] ; then
+        echo "ppc64le/alpine"
+    else
+        echo "unsupport arch"
+        exit
+    fi
+}
+
+get_image_tag() {
+    local current_commit today
+    current_commit="$(git rev-parse HEAD)"
+    today="$(date +%Y%m%d)"
+    echo "v${today}-${current_commit:0:7}"
+}
+
+build_binaries(){
+    # Install bazel
+    apt update && apt install -y curl gnupg git python python3\
```

Comment:
Maybe we should prepare an image with all those tools pre-installed and a certain version of the test-infra repo pre-cloned?

Reply:
> Maybe we should prepare an image with all those tools pre-installed and a certain version of the test-infra repo pre-cloned?

Ok